src/Resources/views/storefront/component/nosto-integration/widget-cart-tagging.html.twig

```diff
@@ -3,7 +3,7 @@
         {% for lineItem in  page.cart.lineItems %}
             {% if lineItem.type == 'product' %}
                 <div class="line_item">
-                    <span class="product_id">{{ lineItem.referencedId }}</span>
+                    <span class="product_id">{{ lineItem.payload.productNumber }}</span>
```

Comment:
is this the case for everyone or just Schlanser? (i just wonder... need to make sure that this doesnt break the functionality for others something like lineItem.referencedId ? lineItem.referencedId : lineItem.payload?.productNumber)?

Reply:
In general, this fix does not pose a problem. I tested it on the Soprex staging shop.